### ISSUE TITLE
NeuronSAT performance on SR3to10.

### ISSUE THREAD

**Issue by lee-man** [ryanzhangfan/NeuroSAT#5]
Hi @ryanzhangfan ,

How about the training accuracy of neuronsat on SR3to10 and test accuracy when it's applied to SR10?
An approximate number would be enough. Thanks!

Min

**Comment by ryanzhangfan** [comment]
Training Set: acc: 0.949, TP: 0.455, TN: 0.494, FN: 0.045, FP: 0.006

**Comment by ryanzhangfan** [comment]
Training Set: acc: 0.949, TP: 0.455, TN: 0.494, FN: 0.045, FP: 0.006

**Comment by ryanzhangfan** [comment]
Training Set (SAT : UNSAT = 1 : 1): acc: 0.971, TP: 0.475, TN: 0.496, FN: 0.025, FP: 0.004
Validation Set (SAT : UNSAT = 1 : 1)：acc: 0.951, TP: 0.456, TN: 0.495, FN: 0.044, FP: 0.005

**Comment by lee-man** [comment]
Thanks a lot! @ryanzhangfan
